examples/jsm/nodes/display/ViewportNode.js

```diff
@@ -93,7 +93,7 @@ class ViewportNode extends Node {
 
 			let coord = builder.getFragCoord();
 
-			if ( builder.isFlipY() ) {
+			if ( ! builder.isFlipY() ) {
```

Comment:
Nit: In general, we have preferred the more explicit style in the project so far:
```js
if ( builder.isFlipY() === false ) {
```
Because it tends to be more readable. Would you mind to adapt this style in the node code base as well?

Reply:
Done!